packages/microcosm-www-next/src/components/graphic1.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```

Comment:
These three graphic.js files (graphic1, graphic2, graphic3) correspond to the image in the left column